spec/validators/dependent_field_inclusion_validator_spec.rb

```diff
@@ -63,4 +63,28 @@ def self.name
       )
     end
   end
+
+  context 'the primary specialism value is missing' do
+    let(:data) { { 'Service Type' => 'Core', 'Primary Specialism' => nil } }
+
+    it { is_expected.to_not be_valid }
+  end
+
+  context 'the primary specialism field is missing' do
+    let(:data) { { 'Service Type' => 'Core' } }
+
+    it { is_expected.to_not be_valid }
+  end
```

Comment:
There's only a need for both these specs (this and the explicit `=> nil` on L68) if we're using `fetch` behaviour, but we're using `dig` in the validator which can return nil without an exception, so I think one of these is redundant.

Reply:
We can poke at this tomorrow in more leisure, as we're not treating it as quite a hotfix anymore.